### ISSUE TITLE
clarification regarding the training set

### ISSUE THREAD

**Issue by tumble-weed** [khrlimam/res-facenet#2]
Hi,
thanks for your work. i am a bit confused about the statement "This model trained on VGGv2 (used the test set one) dataset and tested on LFW dataset and gained 92% accuracy"
Does that mean  the model was trained on VGG Face2 test set?

**Comment by khrlimam** [comment]
> Hi,
> thanks for your work. i am a bit confused about the statement "This model trained on VGGv2 (used the test set one) dataset and tested on LFW dataset and gained 92% accuracy"
> Does that mean the model was trained on VGG Face2 test set?

Yess, you correct. My server was unable to train on VGG train set due to enormous dataset size.
